www/js/app/list/directive/template/movie-list.html

```diff
@@ -0,0 +1,7 @@
+<ion-item class="item-thumbnail-left" href="#/view/{{movieid}}">
+    <img ng-src="{{imageurlw92}}{{posterpath}}"
+         onerror="this.src='img/no_image.png';" />
```

Comment:
Simply because the URL image prefix is static (imageurlw92) and the suffix is dynamic (posterpath).